sensors/imu/mpu6000.c

```diff
@@ -280,7 +282,21 @@ static int mpu6000_alloc(sensor_info_t *info, const char *args)
 	ss = strchr(args, ':');
 	if (ss != NULL) {
 		*(ss++) = '\0';
+
+		lpfChr = strchr(ss, ':');
+		if (lpfChr != NULL) {
+			*(lpfChr++) = '\0';
+
+			errno = EOK;
+			lpfSel = strtoul(lpfChr, NULL, 10);
+			if (lpfSel > 7 || (lpfSel == 0 && errno != EOK)) {
```

Comment:
`lpfSel == 0` is somewhat redundant but it gives performance boost (as no access to `errno` is needed when it's not zero). It causes us to miss ERANGE, but lpfSel > 7 protect against it. So the current code is 100%, but errno check should enough. I won't insist a change here. 
```suggestion
			if ((lpfSel > 7) || (errno != EOK)) {
```

Reply:
You`re right - this can be written shorter. I`ll consider such syntax in future use of `strto*` functions - thanks!. Leaving as is to not force additional changes as already approved :)